src/lib/Sendbird.tsx

```diff
@@ -242,18 +253,32 @@ const Sendbird = ({
             compressionRate: 0.7,
             ...imageCompression,
           },
-          isReactionEnabled,
-          isMentionEnabled: isMentionEnabled || false,
-          isVoiceMessageEnabled,
           voiceRecord,
           userMention: {
             maxMentionCount: userMention?.maxMentionCount || 10,
             maxSuggestionCount: userMention?.maxSuggestionCount || 15,
           },
-          isTypingIndicatorEnabledOnChannelList,
-          isMessageReceiptStatusEnabledOnChannelList,
-          replyType,
           markAsReadScheduler,
+          ...uikitConfigurations,
```

Comment:
Do we need this here with `SendbirdSdkContext`?

Reply:
I left the configs which is being passed to SendbirdSdkContext.Provider for smooth migration. 
Is this what you're asking?

Reply:
IMHO, To maintain the migration by keeping the config of sdkContext in view, it seems safer to elevate UIKitConfigProvider to the top level and pass the values set in UIKitConfigProvider to SendbirdSdkContext through a consumer.

Reply:
What is smoother migration in this situation, btw .. sorry, I am a bit off "context" :D

Reply:
Oh sorry I should've provided more context 😅

So currently we're passing the app level of config to SendbirdSDK provider. But the thing I'm trying to do in this PR is making UIKitConfigProvider handles everything(most of things) related feature configuration instead of SendbirdSDK. 

But I didn't want to make any breaking changes by removing the config values from SendbirdSDK at once. Smoother migration is about this. 

Not sure I gave you a better context ㅋㅋ

Reply:
@bang9 Sorry I missed [your comment](https://github.com/sendbird/sendbird-uikit-react/pull/561#discussion_r1206285328) at the above about the consumer. Just created a PR for it here https://github.com/sendbird/sendbird-uikit-react/pull/572 :)